Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-openshift-4.6 AS builder
```

Comment:
The registry with the `svc` infix is the old registry. We should be using the new one w/out `svc` - e.g. `registry.ci.open...`